content/docs/command-reference/exp/run.md

```diff
@@ -69,27 +69,27 @@ checkpoints with DVC during your code or script runtime (similar to a logger).
 
 To do so, first mark stage `outs` with `checkpoint: true` in `dvc.yaml`. Having
 at least a checkpoint <abbr>output</abbr> is needed so that the experiment can
```

Comment:
```suggestion
To do so, first mark stage `outs` with `checkpoint: true` in `dvc.yaml`.
At least one checkpoint <abbr>output</abbr> is needed so that the experiment can
```

Reply:
Minor style suggestion

Reply:
Applied!